addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -985,6 +987,20 @@ export class Rtc extends Record {
                     }, 2000);
                 }
                 return;
+            case "recovery": {
+                const { id } = payload;
+                const session = this.store["discuss.channel.rtc.session"].get(id);
+                if (!session?.channel.eq(this.state.channel)) {
+                    return;
+                }
+                if (this.serverInfo || this.state.fallbackMode) {
+                    return;
+                }
```

Comment:
all of them could be `||` in a single `if ()`, the nuance is irrelevant since it just aborts the recovery attempt anyway.